RR.UnitOfWork/Entities/HRIS/Employee.cs

```diff
@@ -58,6 +59,44 @@ public Employee(EmployeeDto employeeDto, EmployeeTypeDto employeeType)
         Active = employeeDto.Active;
         InactiveReason = employeeDto.InactiveReason;
         EmployeeType = new EmployeeType(employeeType);
+        LinkedIn = employeeDto.LinkedIn;
+        CV = employeeDto.CV;
+        PortfolioLink = employeeDto.PortfolioLink;
+        PortfolioPDF = employeeDto.PortfolioPDF;
+        Referral = employeeDto.Referral;
+        HighestQualification = employeeDto.HighestQualification;
+        School = employeeDto.School;
+        QualificationEndDate = employeeDto.QualificationEndDate;
+        BlackListStatus = employeeDto.BlackListStatus;
+        BlackListReason = employeeDto.BlackListReason;
+        IsCandidate = employeeDto.IsCandidate;
+    }
+
+    public Employee(CandidateDto candidateDto)
```

Comment:
Shouldn't an Employee construct using an EmployeeDto?

Reply:
I was told to keep the CandidateDto, simply use Employee model. That's why I added a second constructor which takes CandidateDto